pyscriptjs/src/python/pyscript.py

```diff
@@ -124,6 +124,21 @@ def write(element_id, value, append=False, exec_id=0):
         )
 
 
+CURRENT_PY_SCRIPT_TAG = None
+
+
+def set_current_display_target(element):
+    global CURRENT_PY_SCRIPT_TAG
+    CURRENT_PY_SCRIPT_TAG = element
+
+
+def display(value, parent=None, append=True):
+    if parent is None:
+        global CURRENT_PY_SCRIPT_TAG
+        parent = CURRENT_PY_SCRIPT_TAG
+    Element(parent).write(value, append)
```

Comment:
@antocuni I forgot how to declare a global in a class... could you please show me again? :)

Reply:
"a global in a class" -- I don't know what it means :). But maybe you are referring to this pattern:
```python
def set_current_display_target(element):
    get_current_display_target._obj = element

def get_current_display_target():
    return get_current_display_target._obj

get_current_display_target._obj = None
```

I.e., instead of storing it in a global, you store it as an attribute of `get_current_display_target`.